### PR TITLE
ad: Make whole location clickable, fix color

### LINKED ISSUE #52: Only icon inside of 'Lokacija na karti' is clickable
As a user I'd like to click anywhere inside of the action button
As a user, I'd like to have the whole button change color when hovered over

image

### PR DESCRIPTION
**Pull request by Xenosb**
Closes #52

### CODE REVIEW

File: app/views/ads/_ad.html.erb [hunk 1]
```diff
@@ -30,16 +30,16 @@
           </div>
           <% if ad.maps_query.present? %>
             <div class="column is-narrow">
-              <div class="tags has-addons">
-                <span class="tag">Lokacija na karti</span>
-                <span class="tag">
-                  <a href="<%= maps_url(ad.maps_query) %>" target="_blank">
+              <a href="<%= maps_url(ad.maps_query) %>" target="_blank" style="display: block">
```
Comment (Macxim) [inline app/views/ads/_ad.html.erb:33]: We should avoid inline styles.
Please use `is-block` CSS class that comes with Bulma.
See: https://bulma.io/documentation/helpers/visibility-helpers/

**Comment by shime** [comment]
Nice, thanks!

**Review by Macxim — CHANGES_REQUESTED** [review]

**Comment by Macxim** [comment]
Oh too late!

**Comment by shime** [comment]
Oops, sorry about that, @Macxim. There are other instances of using inline styles, like for margin for example. Would you like to fix that?

**Comment by Macxim** [comment]
I definitely will 👍

**Comment by Xenosb** [comment]
Oh 🤦 
It will probably work even without this inline style. I should have double checked my code before PR.

**Comment by Xenosb** [comment]
@Macxim I can fix this right away if you want or you prefer to have a larger PR with the whole cleanup?

**Comment by shime** [comment]
@Macxim will do a complete cleanup in a separate PR.

**Pull request merged**